prometheus_node_exporter.yaml

```diff
@@ -0,0 +1,21 @@
+name: prometheus-node-exporter
+version: 0.16
+summary: Prometheus
+description: |
+  Prometheus is a system monitoring tool. Data available on HTTP port 9100.
```

Comment:
Let's expand this a bit, how about "The node exporter is a system monitoring agent for  the Prometheus monitoring system. ..."